src/include/planner/plannodes/seq_scan_plan_node.h

```diff
@@ -46,6 +46,11 @@ class SeqScanPlanNode : public AbstractScanPlanNode {
      */
     Builder &SetColumnOids(std::vector<catalog::col_oid_t> &&column_oids) {
       column_oids_ = std::move(column_oids);
+      if (column_oids_.empty()) {
+        // TODO(Amadou): In the SqlTable, at least one column must be selected. But Count(*) does not need any column
+        // in particular. Is there a better way than just selecting the first one?
+        column_oids_.emplace_back(1);
```

Comment:
Does this work if that's not a valid `col_oid` anymore? I'm imaging a schema change where maybe we've dropped that column.